test/integration/smoke/test_service_offerings.py

```diff
@@ -1014,6 +999,7 @@ def getHost(self, hostId=None):
 
         )
         cls._cleanup = [
+            cls.vm,
```

Comment:
this will work, but adhering to the structure of letting the parent class do the cleanup won't work this way. Better refactor.